script.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE Speciality 
+(
+	IdSpeciality SERIAL PRIMARY KEY,
```

Comment:
use `id` postfix in the end of column name
Example: `speciality_id`
fix it in all columns